extensions/notebook/src/dialog/configurePython/pickPackagesPage.ts

```diff
@@ -89,45 +124,48 @@ export class PickPackagesPage extends BasePage {
 	}
 
 	public async onPageLeave(): Promise<boolean> {
-		return true;
+		return !this.packageTableSpinner.loading;
```

Comment:
This is to prevent the user from leaving before it's finished loading? Why not just disable the button instead?

Reply:
I already disable the "Install" button when loading, but this is a back up just in case. There's currently a bug where the "Next" button in the wizard doesn't get marked as disabled, even when **enabled** is set to false.

Reply:
Made an issue for that button bug here: https://github.com/microsoft/azuredatastudio/issues/10638